draft-ietf-tls-iana-registry-updates.md

```diff
@@ -217,52 +263,122 @@ TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256   | {0xCC,0xAC}
+IANA [SHALL add/has added] the following note to ensure that those that
+focus on IANA registries are aware that TLS1.3 {{I-D.ietf-tls-tls13}}
+uses the same registry but defines ciphers differently: 
 
-    Note(1): Although TLS 1.3 uses the same cipher suite space as previous versions of TLS, TLS 1.3 cipher suites are defined differently, only specifying the symmetric ciphers, and cannot be used for TLS 1.2. Similarly, TLS 1.2 and lower cipher suites cannot be used with TLS 1.3.
+    Note(1): Although TLS 1.3 uses the same cipher suite space as
+    previous versions of TLS, TLS 1.3 cipher suites are defined
+    differently, only specifying the symmetric ciphers, and cannot be
+    used for TLS 1.2.  Similarly, TLS 1.2 and lower cipher suite values
+    cannot be used with TLS 1.3.
 
-    Note(2): Cipher suites marked as "Yes" are those allocated via Standards Track RFCs.  Cipher suites marked as "No" are not; cipher suites marked "No" range from "good" to "bad" from a cryptographic standpoint.
+IANA [SHALL add/has added] the following notes to document the rules
+for populating the Recommended column:
 
-    Note(3): The designated expert {{RFC5226}} only ensures that the specification is publicly available.
+    Note(2): Cipher suites marked as "Yes" are those allocated via
+    Standards Track RFCs.  Cipher suites marked as "No" are not; cipher
+    suites marked "No" range from "good" to "bad" from a cryptographic
+    standpoint.
+
+    Note(3): The designated expert {{RFC8126}} only ensures that the
+    specification is publicly available.
+
+IANA [SHALL uppdate/has updated] the reference for this registry to
+also refer to this document.
 
 TLS Supported Groups
 ====================
 
-Add a “Recommended” column with a "Yes" for secp256r1, secp384r1, x25519, and x448 while all others are "No".  These "Yes" groups are taken from Standards Track RFCs.  Not all groups from {{I-D.ietf-tls-rfc4492bis}}, which is standards track, are not marked as "Yes"; these groups apply to TLS 1.3 {{I-D.ietf-tls-tls13}} and previous versions of TLS {{I-D.ietf-tls-tls13}}.  A Standards Track document {{RFC5226}} is required to register an entry with the value "Yes".
+Similar to cipher suites, supported groups have proliferated over time
+and some use the registry to measure implementations.  Therefore, IANA
+[SHALL add/has added] a “Recommended” column with a "Yes" for secp256r1,
+secp384r1, x25519, and x448 while all others are "No".  These "Yes"
+groups are taken from Standards Track RFCs.  Not all groups from
+{{I-D.ietf-tls-rfc4492bis}}, which is standards track, are marked as
+"Yes"; these groups apply to TLS 1.3 {{I-D.ietf-tls-tls13}} and
+previous versions of TLS.  Future supported groups MUST define the value
+of this column.  A Standards Track document {{RFC8126}} is required to
+register an entry with the value "Yes".
+
+IANA [SHALL add/has added] the following note:
 
+    Note(2): Supported Groups marked as "Yes" are those allocated via
```

Comment:
The same caution about weakness clearly applies to groups as it does to cipher suites. Why not provide a warning on all registries that identify cryptographic constructs?

Reply:
The only two left I believe are the Hash and Signature algorithm registries that we've orphaned in TLS 1.3?